examples/benchmarks/mappers.py

```diff
@@ -0,0 +1,134 @@
+from thinc.api import remap_ids_v2
+from thinc.api import premap_ids
+from thinc.api import chain, Embed, HashEmbed
+import time
+import random
+import numpy as np
+import cupy as cp
+
+
+N_symbols = 200000
+N_tokens = 50000
+N_batch = 500
+N_columns = 4
+N_dim = 300
+mapper = {}
+numbers = list(range(N_symbols))
+random.shuffle(numbers)
+for v, k in enumerate(numbers):
+    mapper[k] = v
+
+
+def speed_test_no_column():
```

Comment:
Maybe we should start using a test harness?

Reply:
not this PR right? can I resolve this and add a ticket?

Reply:
Yeah, let's do that in a follow-up PR. Though the timings here could use some cleanups. E.g., some of the constructed `start` instances are not used in the tests below. Two other tips:

* It's a very delicate issue, but it's usually better to use `perf_counter` over `process_time`, see e.g.: https://asv.readthedocs.io/en/stable/writing_benchmarks.html#benchmark-types `perf_counter` is also used by e.g. `timeit`.
* I like to time using a context manager. It's cleaner than all the manual timings + computations, see: https://github.com/explosion/spaCy/blob/4c60afb946f35e2675a5e21880ca3a09633d0bfa/spacy/cli/benchmark_speed.py#L60